modules/is/manifests/init.pp

```diff
@@ -46,55 +32,72 @@
     system => true,
   }
 
-  # Ensure /opt/is directory is available
-  file { "/opt/${product}":
-    ensure => directory,
-    owner  => $user,
-    group  => $user_group,
+  # Create distribution path
+  file { [  "${products_dir}",
```

Comment:
Reformat the code

Reply:
https://github.com/wso2/puppet-is/pull/58/commits/b8ba5cf3de87228146971ff913f98bd16021353b